doc/source/examples/analysis/structure/elastic_network.ipynb

```diff
@@ -8,9 +8,9 @@
     "\n",
```

Comment:
<p>As above, this doesn't seem to show much anymore?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Fixed.</p>